server/views/pages/confirm-instructions.njk

```diff
@@ -206,5 +206,4 @@
 {% block javascripts %}
     <script src="/assets/accessible-autocomplete.min.js"></script>
     <script id="autocomplete-script" src="/assets/js/autocomplete-email.js" {%- if scrollToBottom %} data-scroll-to-bottom="true" {% endif %}></script>
-    <script src="/assets/instructions.js"></script>
```

Comment:
This was done as part of the `instructions` macro

Reply:
I vaguely remember this block replaces the definition in the `instructions` macro rather than merges both together, we can see what it does in dev whether it brings in the script